plugin/trino-pulsar/src/main/java/io/trino/plugin/pulsar/PulsarAdminClientProvider.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pulsar;
+
+import org.apache.pulsar.client.admin.PulsarAdmin;
+import org.apache.pulsar.client.admin.PulsarAdminBuilder;
+import org.apache.pulsar.client.api.PulsarClientException;
+
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.client.ClientBuilder;
+
+public class PulsarAdminClientProvider
+{
+    private PulsarAdminClientProvider()
+    { }
+
+    @NotNull
```

Comment:
In Trino everything by default is not null. We annotate only `@Nullable` variables.

Reply:
done